### PR TITLE
Update docs from ROS feedback

### PR DESCRIPTION
**Pull request by ernesgonzalez33**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +80,6 @@ namespace: <your-workspace-name>-tenant
 ```
 5. Run the `build-manifests.sh`. This is script is using [Kustomize](https://kustomize.io/) to generate the Integration Test Scenario and secrets you will need to run the pipeline in the cluster. Check that the `auto-generated` directory is updated with these files. 
 6. Commit your directory and the `auto-generated` directory.
-7. Create a PR and ask for approval in the [#rhtap-users](https://redhat-internal.slack.com/archives/C04PZ7H0VA8) Slack channel.
+7. Create a PR from your fork, and ask for approval in the [#rhtap-users](https://redhat-internal.slack.com/archives/C04PZ7H0VA8) Slack channel.
```
Comment (dfodorRH) [inline README.md:83]: do people in #rhtap-users have permissions and knowledge to approve in https://github.com/gbenhaim/tekton-insights ?
maybe suggest to also mention @ergonzal or @gbenhaim in the thread

Comment (ernesgonzalez33) [reply README.md:83]: The forked repository is not tekton-insights. It is https://github.com/redhat-appstudio/tenants-config. I think there should be more people that can approve there. But I don't mind telling them to mention us

Comment (ernesgonzalez33) [reply README.md:83]: @gbenhaim WDYT?

Comment (gbenhaim) [reply README.md:83]: right now we (migration) and the release team can approve prs in that repo.

**Comment by gbenhaim** [comment]
@ernesgonzalez33 how can we make progress with this PR?

**Comment by ernesgonzalez33** [comment]
@gbenhaim If you are OK with the content, I can solve the conflicts tomorrow and merge it.

**Pull request merged**